opentech/apply/funds/templates/funds/includes/update_lead_form.html

```diff
@@ -0,0 +1,5 @@
+<form method="post" id="update-lead-form">
+    {% csrf_token %}
+    {{ lead_form }}
```

Comment:
Would be good set the default value to the current lead, if any

Reply:
I explicitly removed the current lead from the list. This was to prevent issues where you reassign the project back to current lead.

Perhaps we could update the label to be: "Update lead from {{ current lead }} to"